### PR TITLE
Update urdf.py to have tag on joint and link reflections

### PR DESCRIPTION
**Pull request by willcbaker**
Enables user to use to_xml() function on joints and links

### CODE REVIEW

**Comment by clalancette** [comment]
The concept seems OK to me; I don't think this will break any backwards compatibility, either.  However, if we are going to go down the route of adding tags, I'd rather we do it for all of the reflections in urdf.py.  @sloretz , thoughts?

File: src/urdf_parser_py/urdf.py [hunk 1]
```diff
@@ -301,7 +301,7 @@ def joint_type(self): return self.type
 	@joint_type.setter
 	def joint_type(self, value): self.type = value
 
-xmlr.reflect(Joint, params = [
+xmlr.reflect(Joint, tag = 'joint', params = [
```
Comment (sloretz) [inline src/urdf_parser_py/urdf.py]: nitpick, no spaces next to the equals sign in keyword argument.

File: src/urdf_parser_py/urdf.py [hunk 2]
```diff
@@ -324,7 +324,7 @@ def __init__(self, name=None, visual=None, inertial=None, collision=None, origin
 		self.collision = collision
 		self.origin = origin
 
-xmlr.reflect(Link, params = [
+xmlr.reflect(Link, tag = 'link', params = [
```
Comment (sloretz) [inline src/urdf_parser_py/urdf.py]: Same, There should be no spaces next to equal sign. Of course, nothing else in this module follows pep 8, but it'd be nice to start.

**Review by sloretz — CHANGES_REQUESTED** [review]
@willcbaker Adding the tag is a great idea. Just a tiny formatting change is needed.

@clalancette yeah I agree. If one tag supports a feature, they all should. @willcbaker would you mind adding tags to the ones which don't already have it?

**Comment by clalancette** [comment]
@willcbaker Given that this seems to be subsumed by #20, closing this.  If I'm mistaken, feel free to reopen.

**Pull request closed without merging**